### PR TITLE
[zh-CN]: sync translation for Fullscreen API events

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

[Document: fullscreenchange](https://developer.mozilla.org/en-US/docs/Web/API/Document/fullscreenchange_event)
[Document: fullscreenerror](https://developer.mozilla.org/en-US/docs/Web/API/Document/fullscreenerror_event)
[Element: fullscreenchange](https://developer.mozilla.org/en-US/docs/Web/API/Element/fullscreenchange_event)
[Element: fullscreenerror](https://developer.mozilla.org/en-US/docs/Web/API/Element/fullscreenerror_event)

### Related issues and pull requests

part of the https://github.com/mdn/translated-content/issues/23557

~depends on https://github.com/mdn/content/pull/36012~

### CODE REVIEW

File: files/zh-cn/web/api/element/fullscreenchange_event/index.md [hunk 1]
```diff
@@ -1,53 +1,82 @@
...
+```js
+addEventListener("fullscreenchange", (event) => {});
+
+onfullscreenchange = (event) => {};
 ```
 
-### 值
+## 事件类型
+
+通用的 {{domxref("Event")}} 类型
```
Comment (yin1999) [inline files/zh-cn/web/api/element/fullscreenchange_event/index.md:30]: 统一

```suggestion
通用的 {{domxref("Event")}}。
```

File: files/zh-cn/web/api/element/fullscreenchange_event/index.md [hunk 2]
```diff
@@ -1,53 +1,82 @@
...
+## 示例
 
-## Example
+在此示例中，向 ID 为 `fullscreen-div` 的元素添加了 `fullscreenchange` 事件处理器。
 
-本示例建立一个`fullscreenchange` 处理程序，`handleFullscreenChange ()`。此函数通过检查 [`event.target`](/zh-CN/docs/Web/API/Event/target) 的值来确定调用它的元素，然后将文档的[`fullscreenElement`](/zh-CN/docs/Web/API/Document/fullscreenElement) 值与元素进行比较，以查看它们是否为同一节点。
+如果用户单击“切换全屏模式”按钮，`click` 处理器将切换 `div` 的全屏模式。如果 `document.fullscreenElement` 具有值，则会退出全屏模式。否则，div 将被置于全屏模式。
 
-这给了我们一个值，即 `isFullscreen`, 我们将其传递到一个名为 `adjustMyControls()` 的函数，我们想象它是一个函数，可以对应用的用户界面进行调整，以便在全屏模式下而不是在窗口。
+请记住，在处理 `fullscreenchange` 事件时，元素的状态已经发生了变化。因此，如果更改是为了全屏模式，`document.fullscreenElement` 将指向现在处于全屏模式的元素。另一方面，如果 `document.fullscreenElement` 为 `null`，则全屏模式已被取消。
```
Comment (yin1999) [inline files/zh-cn/web/api/element/fullscreenchange_event/index.md:38]: 感觉这样翻译更好（to 表示要发生的动作）

```suggestion
请记住，在处理 `fullscreenchange` 事件时，元素的状态已经发生了变化。因此，如果更改是进入全屏模式，`document.fullscreenElement` 将指向现在处于全屏模式的元素。另一方面，如果 `document.fullscreenElement` 为 `null`，则全屏模式已被取消。
```

File: files/zh-cn/web/api/element/fullscreenchange_event/index.md [hunk 3]
```diff
@@ -1,53 +1,82 @@
...
+
+对于示例代码，这意味着，如果元素当前处于全屏模式，则 `fullscreenchange` 处理器在控制台中打印全屏元素的 `id`。如果 `document.fullscreenElement` 为 `null`，则代码记录消息表示更改是为了退出全屏模式。
```
Comment (yin1999) [inline files/zh-cn/web/api/element/fullscreenchange_event/index.md:40]:
```suggestion
对于示例代码，这意味着，如果元素当前处于全屏模式，则 `fullscreenchange` 处理器在控制台中打印全屏元素的 `id`。如果 `document.fullscreenElement` 为 `null`，则代码记录消息表示更改是退出全屏模式。
```

**Review by yin1999 — APPROVED** [review]
:+1:

**Comment by q1763982** [comment]
thank,you

**Pull request merged**